src/olympia/pages/templates/pages/shield_study_5.html

```diff
@@ -0,0 +1,67 @@
+        </div>
+        <div class="content"><h1 id="context-graph-experiment">Context Graph Experiment</h1>
+<p>Mozilla wants to test the viability of using information about the
+sites users visit to 
+<a href="https://medium.com/@osunick/context-graph-its-time-to-bring-context-back-to-the-web-a7542fe45cf3">build a recommendation engine that improves discovery on the web</a>.</p>
+<p>Participating in this study will help us find out!</p>
+<h2 id="what-will-happen-next">What will happen next</h2>
+<p><strong>If you agree</strong> to participate:</p>
+<ol>
+<li>We will add additional features to Firefox that record the URLs you visit and send that data to our server. This data will be anonymous and not connected to other information about you.</li>
+<li>Continue using Firefox as you normally would. You will not notice any changes.</li>
+<li>Within three months, the study will end on its own. Your Firefox will
+go back to normal.</li>
+<li>We will ask you to fill out a quick optional survey to tell us about your experience. The survey should take less than 5 minutes to complete.</li>
+</ol>
+<h2 id="leaving-the-study">Leaving the study</h2>
+<p>The study will expire on its own within three months. You may leave
+the study early. To do so, follow these steps:</p>
+<ol>
+<li>Type about:addons into the location bar, and press enter.</li>
+<li>Find Context Graph Recommendation Engine Experiment in the addons list.</li>
+<li>Click the remove button.</li>
+<li>If you opt out of the study early we will ask you to fill out a survey. We’re interested in hearing about your experience, even if you didn’t participate in the entire study.</li>
+<li>Opting out of the study early will delete the data you submitted to the server.</li>
+</ol>
+<p>Opting out of a study does not prevent you from participating in future studies.</p>
+<h2 id="your-privacy">Your privacy</h2>
+<p>Here are the key things you should know about what data is collected
+when you participate in this study:</p>
+<ul>
+<li>We collect URLs you visit, which tab you were in, when you visited and
+how long you spent on the page. We filter some potentially sensitive
+URLs and edit others to improve anonymity.</li>
+<li>This data will be linked to a random ID we only use for this
+experiment. It is used to keep a specific user&#39;s data together on the
+server and to allow a user to request that their data be deleted. </li>
+<li>Important study events, such as install, uninstall, and end of study
+will be collected.</li>
+</ul>
+<p>Our practices for collecting and handling the data can be found <a href="https://github.com/mozilla/heatmap/blob/master/docs/data_practices.md">here</a>.</p>
+<p>Technical specifications about the data collection for this study are
+<a href="https://github.com/mozilla/heatmap/blob/master/docs/submit.rst">here</a>.</p>
```

Comment:
For improved accessibility in the future it would be better to link more than just "here" so that the link itself contains more context.